### PR TITLE
Feature/p22-25 security

### PR DESCRIPTION
**Pull request by milicasamardzija**

### CODE REVIEW

File: Tennis-scheduler/pom.xml [hunk 1]
```diff
@@ -26,7 +26,9 @@
 		<spring-boot-starter-web.version>2.7.4</spring-boot-starter-web.version>
 		<springdoc-openapi-ui.version>1.6.11</springdoc-openapi-ui.version>
 		<spring-boot-starter-test.version>2.7.4</spring-boot-starter-test.version>
+		<jwt.version>0.6.0</jwt.version>
 		<h2.version>2.1.214</h2.version>
+    <swagger-annotations.version>2.2.2</swagger-annotations.version>
```
Comment (markoacimovic) [inline Tennis-scheduler/pom.xml]: Please indent 😄

File: Tennis-scheduler/pom.xml [hunk 2]
```diff
@@ -96,6 +104,17 @@
 			<artifactId>mapstruct</artifactId>
 			<version>${mapstruct.version}</version>
 		</dependency>
+		<!-- ukljucena podrska za Spring Security -->
```
Comment (markoacimovic) [inline Tennis-scheduler/pom.xml]: On English comments

**Review by Dzela — APPROVED** [review]

File: Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/services/PersonService.java [hunk 3]
```diff
@@ -15,6 +17,8 @@
 public class PersonService {
 
     private final PersonRepository personRepository;
+    private final RoleRepository roleRepository;
+    private final PasswordEncoder bCryptPasswordEncoder;
```
Comment (markoacimovic) [inline Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/services/PersonService.java:21]: I think you have to explicitly import BCrypt to use it.

Comment (markoacimovic) [reply Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/services/PersonService.java:21]: Just check this, because i am not 100% sure

Comment (milicasamardzija) [reply Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/services/PersonService.java:21]: Is it enough that i specifed in spring security config implemenantion of password encoder with bean annotation (that is BCrypt)?I'm not sure, but when i use debug it looks like it use that implemenatition.

image

**Review by nemanjaromanic — APPROVED** [review]

**Review by mirkovicjul — APPROVED** [review]

**Review by markoacimovic — APPROVED** [review]

**Pull request merged**